code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -116,3 +116,17 @@
 	new /obj/item/stack/cable_coil(src,30,color)
 	new /obj/item/weapon/wirecutters(src)
 	new /obj/item/device/multitool(src)
+
+/obj/item/weapon/storage/toolbox/robotics
+	name = "robotics toolbox"
+
+/obj/item/weapon/storage/toolbox/robotics/New()
+	..()
+	var/color = pick("red","yellow","green","blue","pink","orange","cyan","white")
```

Comment:
Use a standard colour for crying out loud

Reply:
why though?

Reply:
Ooh, I misread and thought it was the colour of the toolbox.

Been a rough week, my apologies

Reply:
I'm pretty sure there's a /obj/item/stack/cable_coil/random

Reply:
nvm this entire file is done this way, oh well